config/runtime/fstests.jinja2

```diff
@@ -24,6 +24,21 @@ class Job(BaseJob):
             print('Kernel config error:', e)
             return 'fail'
 
+    def _kernel_build(self, kdir):
+        try:
+            os.chdir(kdir)
+            result = subprocess.run("make -j$(nproc)", check=True, shell=True)
+            if result.returncode == 0:
+                if os.path.exists('arch/x86/boot/bzImage'):
```

Comment:
@gctucker do we need to check if we have an image? I mean, in case of any problem we will get an exception..

Reply:
I don't think we need to check for the presence of the kernel image.